tests/Integration/PrestaShopBundle/Command/LoadLegacyClassesinCommandTest.php

```diff
@@ -52,7 +52,13 @@ public function setUp()
 
     public function testLoadLegacyCommandWithoutContextFails()
     {
-        $this->expectException(TypeError::class, 'Not enough arguments (missing: "theme, locale").');
+        if (version_compare(phpversion(), '8.0', '>=')) {
+            $this->expectWarning();
+            $this->expectWarningMessage('Attempt to read property "precision" on null');
```

Comment:
Are we trying to hide a real error? :thinking:

Reply:
I guess you discussed this, but can you explain here the reason for this change for the buddies? ^^
Or maybe add a comment to explain the difference between the two versions

Reply:
FYI, expectException doesn't have a second parameter, it only checks for TypeError. But @atomiix will explain better than me why the message is different